kuksa-sdk/src/test/kotlin/org/eclipse/kuksa/vssSpecification/VssSpecificationCopyTest.kt

```diff
@@ -58,7 +58,7 @@ class VssSpecificationCopyTest : BehaviorSpec({
             val invertedSpecification = !vehicle.driver.isEyesOnRoad
 
             then("it should return a copy with the inverted value") {
-                invertedSpecification.value shouldBe false
+                invertedSpecification.value shouldBe true
```

Comment:
The default value from VssIsEyesOnRoad changed from true to false, therefore obviously the inverse needs to be inversed :)